lib/google_maps_geocoder/google_maps_geocoder.rb

```diff
@@ -75,7 +75,7 @@ def initialize(address, logger: Logger.new($stderr))
     @json = address.is_a?(String) ? google_maps_response(address) : address
     status = @json && @json['status']
     raise RuntimeError if status == 'OVER_QUERY_LIMIT'
-    raise GeocodingError, @json if !@json || @json.empty? || status != 'OK'
+    raise GeocodingError.new(@json, logger: logger) if !@json || @json.empty? || status != 'OK'
```

Comment:
https://blog.saeloun.com/2021/09/28/ruby-allow-value-omission-in-hash-literals/

```suggestion
    raise GeocodingError.new(@json, logger:) if !@json || @json.empty? || status != 'OK'
```

Reply:
Right on. I still haven't grokked the new arg syntax.